sherpa/sherpa-standalone.rc

```diff
@@ -22,6 +22,11 @@ trunc_value: 1.0e-25
 minimum_energy: 1.0e-10
 
 [verbosity]
+# Sherpa Chatter level
+# a non-zero value will
+# display full error traceback
+# This option is only used by Sherpa up to version 4.10.1
```

Comment:
Technically (e.g. in case of a 4.10.2 release) this - and the version in `sherpa.rc` - should be more like

```
This option is not used by Sherpa 4.11 but is left in so as to not cause problems
with systems that may switch between different Sherpa versions (see #490)
```

Even if we fix #490 we can't remove these settings until the pre-#490 fixed code is no-longer in common use.

I didn't change the PR in case this is "too much" and because I didn't want to start a new set of travis tests.